landingai/common.py

```diff
@@ -34,6 +34,10 @@ class Prediction(BaseModel):
     score: float
     """The confidence score of this prediction."""
 
+    def num_predicted_pixels(self) -> int:
```

Comment:
for consistency between child classes, i think we need the property decorator
```suggestion
    @property
    def num_predicted_pixels(self) -> int:
```

Reply:
done